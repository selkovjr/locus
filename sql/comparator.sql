--
--  Locus datatype test
--
-- Testing the comparator
--
select locus_cmp('chr6', 'chr21');
select locus_cmp('chr6:29474946-29475446'::locus, 'chr21:29474869-29475900');
select locus_cmp('chr6:29474946-29475446'::locus, 'chr6:28474869-29475900');
