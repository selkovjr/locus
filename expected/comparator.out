--
--  Locus datatype test
--
-- Testing the comparator
--
select locus_cmp('chr6', 'chr21');
 locus_cmp
-----------
        -1
(1 row)

select locus_cmp('chr6:29474946-29475446'::locus, 'chr21:29474869-29475900');
 locus_cmp
-----------
        -1
(1 row)

select locus_cmp('chr6:29474946-29475446'::locus, 'chr6:28474869-29475900');
 locus_cmp
-----------
         1
(1 row)

